docs/install-pro.md

```diff
@@ -0,0 +1,97 @@
+# Install Percona XtraBackup Pro
+
+--8<--- "pro-build-announcement.md"
+
+This document provides guidelines how to install Pro packages of Percona XtraBackup from Percona repositories. [Check files in packages built for Percona XtraBackup Pro :material-arrow-right:](pro-files.md){.md-button}
+
+## Procedure
+
+1. Request the access to the pro repository from Percona Support. You will receive the client ID and the access token which you use when downloading the packages.
+
+3. Configure the repository and install Percona XtraBackup packages
+
+    === "On Debian or Ubuntu"
+
+        1. Download a `DEB` package for *percona-release* the repository packages from Percona web:
+
+            ```{.bash data-prompt="$"}
+            $ wget https://repo.percona.com/apt/percona-release_latest.$(lsb_release -sc)_all.deb
+            ```
+
+        2. Install the downloaded package with **dpkg**. To do that, run the following commands as root or with **sudo**: `dpkg -i percona-release_latest.$(lsb_release -sc)_all.deb`
+   
+            Once you install this package the Percona repositories should be added. You
+            can check the repository setup in the `/etc/apt/sources.list.d/percona-release.list` file.
+
+        3. Enable the repository: `percona-release enable-only tools release`
+
+            If Percona XtraBackup is intended to be used in combination with the upstream MySQL Server, you only need to enable the `tools` repository: `percona-release enable-only tools`.
+
+        4. Refresh the local cache to update the package information:
+
+            ```{.bash data-prompt="$"}
+            $ sudo apt update
+            ```
+
+        5. Install the `percona-xtrabackup-pro` package:
+
+            ```{.bash data-prompt="$"}
+            $ sudo apt install percona-xtrabackup-pro
```

Comment:
I see the package name is "percona-xtrabackup-pro-84"

`sudo apt-cache search percona-xtrabackup
percona-xtrabackup-pro-84 - Open source backup tool for InnoDB and XtraDB
percona-xtrabackup-pro-dbg-84 - Debug symbols for Percona XtraBackup
percona-xtrabackup-pro-test-84 - Test suite for Percona XtraBackup
percona-xtrabackup-test-pro-84 - Test suite for Percona XtraBackup
percona-xtrabackup-84 - Open source backup tool for InnoDB and XtraDB`

sudo apt install percona-xtrabackup-pro
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package percona-xtrabackup-pro

Reply:
renamed